app/workers/github/create_issue_job.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+require 'github/github_service'
+
+module Github
+  class CreateIssueJob
+    include Sidekiq::Worker
+
+    # limit the rate at which Github API calls are made
+    # so as not to trigger Github spam protections
+    sidekiq_options rate: {
```

Comment:
Although it's not list in the vets-api procfile, in [prod there's a 'low' queue](https://github.com/department-of-veterans-affairs/devops/blob/9bf1948a2005cf46aae2ba55d603c0196d0efc00/ansible/roles/vets-api-worker/meta/main.yml#L12), we could use it for this job so this isn't on the same queue as higher priority ones.